cdp4common/src/main/java/cdp4common/helpers/PojoThingFactory.java

```diff
@@ -192,6 +192,11 @@ public static void resolveDependencies(Iterable<cdp4common.dto.Thing> dtoThings,
      * @return The {@link Thing} of type {@link T}
      */
     public static <T extends Thing> T get(Cache<CacheKey, Thing> cache, UUID itemIid, UUID iterationId, Class<T> clazz) {
+        if(itemIid == null){
+            log.debug(String.format("The %1$s is not possible to found in the cache for item: %2$s", clazz.getSimpleName(), itemIid));
```

Comment:
please rephrase the text of the log to: "The unique identifier xxx could not be found in the Cache for item: yyy". If the text in the C# SDK needs to be updated as well, then please post a GH issue so we harmonize